conference.js

```diff
@@ -650,6 +655,14 @@ export default {
             }, NOTIFICATION_TIMEOUT_TYPE.LONG));
         }
 
+        if ('PressureObserver' in window.globalThis) {
```

Comment:
This should be in a middleware, not tied to the existence of the meeting, but the whole app. Something like APP_WILL_MOUNT perhaps.

Also, even if calling observe returns a promsie there is no good reason to make this method async because we don't care about the result, we could just use `.catch` and log an error if necessary.